src/App.js

```diff
@@ -16,7 +17,7 @@ function App() {
 
       <SignUp path="/sign-up" />
       <SignIn path="/sign-in" />
-
+      <SignUpResearcher path="/sign-up-researcher" />
```

Comment:
Hmm is it better to change this route to `/sign-up/researcher` and to incorporate the route for volunteers as `/sign-up/volunteer`. These two routes can be nested under `/sign-up`. 

The code can look something like:
```javascript 
<SignUp path="/sign-up/*" />
```
in **App.js** and in the `SignUp` component, after the user clicks on `Next`:
```javascript 
<Router>
    <SignUpResearcher path="researcher" />
    <SignUpVolunteer path="volunteer" />
</Router>
```
they can be directed based on user types. 

The **Reach Router** (https://reach.tech/router/) is used here which is slightly different from **React Router**. Do you prefer **React Router** instead? 😮

Reply:
added reach router in the latest commit. Cant seem to make nested routing work so normal routing was used instead. Maybe I'll try it out if theres spare time !

Reply:
okies sure!! 👍